jekyll/_cci2/hello-world.adoc

```diff
@@ -0,0 +1,531 @@
+
+[source,yaml]
+----
+version: 2.1
+
+jobs:
+  hello-job:
+    macos:
+      xcode: 13.4.1
+    steps:
+      - checkout # check out the code in the project directory
+      - run: echo "hello world" # run the `echo` command
+
+workflows:
+  my-workflow:
+    jobs:
+      - hello-job
+----
+--
++
+[.tab.hellocloud.Windows]
+--
+The job `hello-job` spins up a Windows virtual machine using the default executor specified by the link:https://circleci.com/developer/orbs/orb/circleci/windows#usage-run_default[Windows orb]. Refer to link:/docs/using-windows[Using the Windows execution environment] page for more information.
+
+[source,yaml]
+----
+version: 2.1
+
+orbs:
+  win: circleci/windows@5.0.0 # The Windows orb gives you everything you need to start using the Windows executor.
+
+jobs:
+  hello-job:
+    executor:
+      name: win/default # executor type
+      size: "medium" # resource class, can be "medium", "large", "xlarge", "2xlarge", defaults to "medium" if not specified
+
+    steps:
+      # Commands are run in a Windows virtual machine environment
+      - checkout
+      - run: Write-Host 'Hello, Windows'
+
+workflows:
+  my-workflow:
+    jobs:
+      - hello-job
+----
+--
++
+[.tab.hellocloud.GPU]
+--
+NOTE: The GPU execution environment is available on the link:https://circleci.com/pricing/[Scale] plan.
+
+The job `hello-job` spins up a GPU-enabled virtual machine using the machine executor. GPU images are available for link:/docs/configuration-reference#available-windows-gpu-image[Windows] and link:/docs/configuration-reference#available-linux-gpu-images[Linux]. Refer to link:/docs/using-gpu[Using the GPU execution environment] page for more information.
+
+[source,yaml]
+----
+version: 2.1
+
+jobs:
+  hello-job:
+    machine:
+      image: ubuntu-2004-cuda-11.4:202110-01
+    steps:
+      - checkout # check out the code in the project directory
+      - run: echo "hello world" # run the `echo` command
+
+workflows:
+  my-workflow:
+    jobs:
+      - hello-job
+----
+--
++
+[.tab.hellocloud.Arm]
+--
+The job `hello-job` spins up an [Arm (Linux) virtual machine] using the machine executor. Refer to link:/docs/using-arm[Using the Arm execution environment] page for more information.
+
+[source,yaml]
+----
+version: 2.1
+
+jobs:
+  hello-job:
+    machine:
+      image: ubuntu-2004:202101-01
+    resource_class: arm.medium
+    steps:
+      - checkout # check out the code in the project directory
+      - run: echo "hello world" # run the `echo` command
+
+workflows:
+  my-workflow:
+    jobs:
+      - hello-job
+----
+--
+
+. Commit and push the changes to your VCS.
+
+. Go to the **Projects** page in the CircleCI web app, then click the **Set Up Project** button next to your new project. If you do not see your project, make sure you have selected the correct organization at the top-left of the web app. See the link:/docs/introduction-to-the-circleci-web-app#organizaiton-switching[Organization Switching] for more information.
```

Comment:
```suggestion
. Go to the **Projects** page in the CircleCI web app, then click the **Set Up Project** button next to your new project. If you do not see your project, make sure you have selected the correct organization at the top-left of the web app. See the link:/docs/introduction-to-the-circleci-web-app#organization-switching[Organization Switching] for more information.
```
"Organization Switching page"? Also do we stick to title case or sentence case?

Reply:
I'll stick with title case for now but if/when we decide to change we can hopefully work out a way to make a global change